### PR TITLE
release v0.2.0

### PR DESCRIPTION
**Pull request by freeznet**
*(If this PR fixes a github issue, please add `Fixes #<xyz>`.)*

Fixes #<xyz>

*(or if this PR is one task of a github issue, please add `Master Issue: #<xyz>` to link to the master issue.)*

Master Issue: #<xyz>

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve.*

### Modifications

*Describe the modifications you've done.*

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

Check the box below.

Need to update docs? 

- [ ] `doc-required` 
  
  (If you need help on updating docs, create a doc issue)
  
- [ ] `no-need-doc` 
  
  (Please explain why)
  
- [x] `doc` 
  
  (If this PR contains doc changes)

### CODE REVIEW

**Review by tpiperatgod — APPROVED** [review]
lgtm

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
+# Function Mesh v0.2.0 Release Notes
+
+## v0.2.0 What's New
+
+- [Helm Charts] Release function-mesh-operator to helm charts repository ([#356](https://github.com/streamnative/function-mesh/issues/356))
```
Comment (Huanli-Meng) [inline CHANGELOG.md:5]:
```suggestion
- [Helm Charts] Release the FunctionMesh operator to helm charts repository ([#356](https://github.com/streamnative/function-mesh/issues/356))
```

**Review by Huanli-Meng — APPROVED** [review]
Only one minor comment.

**Comment by nlu90** [comment]
👍🏼

**Pull request merged**